app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit/Tab/Super/Config/Simple.php

```diff
@@ -108,7 +110,9 @@ protected function _prepareForm()
         }
 
         /* Configurable attributes */
-        $usedAttributes = $this->_getProduct()->getTypeInstance(true)->getUsedProductAttributes($this->_getProduct());
+        /** @var Mage_Catalog_Model_Product_Type_Configurable $productType */
+        $productType = $this->_getProduct()->getTypeInstance(true);
+        $usedAttributes = $productType->getUsedProductAttributes($this->_getProduct());
```

Comment:
Just asking, is re-declaring `$productType` and `$usedAttributes` for BC? I do not see how the values of these two vars can changed from lines 62-63.

Reply:
I just fixed the lines mentioned at phpstan baseline. I did not check for duplicate code. :+1: